Shaders/Materials/BlinnPhong/LitOITBlinnPhong.frag.glsl

```diff
@@ -3,14 +3,16 @@ layout (location = 1) out vec4 f_Revealage;
 
 #include "TransformStructs.glsl"
 #include "DefaultLight.glsl"
-#include "BlinnPhongMaterial.glsl"
 
 layout (location = 0) in vec3 in_position;
 layout (location = 1) in vec3 in_texcoord;
 layout (location = 2) in vec3 in_normal;
 layout (location = 3) in vec3 in_tangent;
 layout (location = 4) in vec3 in_viewVector;
 layout (location = 5) in vec3 in_lightVector;
+layout (location = 6) in vec3 in_vertexcolor;
+
+#include "BlinnPhongMaterial.glsl"
```

Comment:
why that change?

Reply:
Because a bad BSDF interface design (blend between varying and uniform parameters in some BSDF related functions). To be corrected in V2.